### ISSUE TITLE
cepton_export.py overwrites the output_dir

### ISSUE THREAD

**Issue by kazuf3** [ceptontech/cepton_sdk_redist#11]
What I thought: it will generate files into `output_dir`, while if it fail, it deletes everything in `output_dir`.
Is this a documented behavior?

**Comment by kazuf3** [comment]
https://github.com/ceptontech/cepton_sdk_redist/blob/50995c771643f73b445a2b13d8010d2de735202c/python/tools/cepton_export.py#L74-L75

this does what exactly it does.

**Comment by liaody** [comment]
Thanks reporting this bug, we have a fix that will be included in the next release:
```diff
-  shutil.rmtree(output_dir, ignore_errors=True)
-  os.makedirs(output_dir)
+  os.makedirs(output_dir, exist_ok=True)
```

**Comment by liaody** [comment]
Fix released in v1.17.7
